woodwork/statistics_utils.py

```diff
@@ -306,3 +338,79 @@ def _get_value_counts(dataframe, ascending=False, top_n=10, dropna=False):
         values = list(df.to_dict(orient="index").values())
         val_counts[col] = values
     return val_counts
+
+
+def _get_top_values_numeric(series, _range):
+    """Get the most frequent values in a range of integers. Will ignore
+        null values.
+
+    Args:
+        series (pd.Series): data to use to find most frequent values
+        _range (type(range)): sequence of integers for which to count occurrences in series
+
+    Returns:
+        top_list (list(dict)): a list of dictionary with keys `count` and
+            `value`
+    """
+    frequencies = series.value_counts(dropna=True)
+    return [
+        {"count": frequencies[i] if i in frequencies else 0, "value": i} for i in _range
+    ]
+
+
+def _get_top_values_categorical(series, num_x):
+    """Get the most frequent values in a pandas Series. Will exclude null values.
+
+    Args:
+        column (pd.Series): data to use find most frequent values
+        num_x (int): the number of top values to retrieve
+
+    Returns:
+        top_list (list(dict)): a list of dictionary with keys `count` and
+            `value`
```

Comment:
Probably worth mentioning that the list sorted so that the later in the list, the higher the count (I initially assumed the opposite, and since the dataframe output only shows the first element of the list, was confused as to why it didn't match the value at mode).

Either that, or the order should be reversed.

Reply:
Also, I think that the numeric top count is sorted on `value` and not `count`, so they should probably match or be more clearly differentiated.

Reply:
Another small inconsistency is that the dict here is `{value:n, count:n}`, and in the numeric calculation its `{count:n, value:n}`

Reply:
Left sorting order the same for now, but updated all dictionaries so the are formatted as `{"value": n, "count": x}`

Reply:
Also updated docstrings to mention the sorting order of the output.